### PR TITLE
Fix: Change portal language

### PR DESCRIPTION
**Pull request by Bilelkihal**
Related issue: https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/863

Changes: 
- Fix a wrong condition done in this PR: https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/748

So now the default behavior will work, while keeping the preventing of selecting a disabled language like German for example.

Screenshots:
image
image

### CODE REVIEW

**Review by Bilelkihal — COMMENTED** [review]
✅

**Comment by Bilelkihal** [comment]
Deployed to stage

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def set_locale
     cookies.permanent[:locale] = I18n.locale if cookies[:locale].nil?
     logger.debug "* Locale set to '#{I18n.locale}'"
 
-    I18n.locale = portal_lang if portal_language_enabled?(I18n.locale)
+    I18n.locale = portal_lang unless portal_language_enabled?(I18n.locale)
```
Comment (syphax-bouazzouni) [inline app/controllers/application_controller.rb:30]: How do you explain this change?

Comment (Bilelkihal) [reply app/controllers/application_controller.rb:30]: we set `I18n.locale` to `portal_lang` which is in our case `en` only when the `I18n.locale` is not enabled, when it's enabled we do nothing we keep the previous behavior (change the language of the portal).

So now if you do manually:
`https://stageportal.lirmm.fr/locale/de` for example then `I18n.locale` is not enabled, we use `portal_lang` which is `en` so the portal keep using english

but changing to french for example, the portal language is enabled so we do nothing and we change the portal language to french.

It's deployed to stage, you can test it.

**Review by syphax-bouazzouni — APPROVED** [review]

**Pull request merged**